newdle/client/src/components/NewdlesParticipating.js

```diff
@@ -24,7 +25,9 @@ export default function NewdlesParticipating() {
   } else if (participations.length === 0) {
     content = (
       <div className={styles['no-newdle-container']}>
-        <h2>You are not part of any newdles yet.</h2>
+        <Trans>
+          <h2>You are not part of any newdles yet.</h2>
```

Comment:
Put the trans inside the h2 - no need to include the tag placeholders in the translatable string for cases where they are just surrounding the whole string

Reply:
Oh yea. oops... Fixed. Thanks!